### ISSUE TITLE
kube 1.16 support - no matches for kind "StatefulSet" in version "apps/v1beta2"

### ISSUE THREAD

**Issue by drnic** [cloudfoundry-incubator/cf-operator#674]
I tried installing `v0.4.2-dirty+85.gc6d71da5` on kube 1.16.2 via helm2/tiller and it failed with:

```
2019/10/31 00:03:34 cf-operator command failed. Failed to create new manager.: failed to add controllers to manager: Watching Statefulset failed in Statefulset cleanup controller.: no matches for kind "StatefulSet" in version "apps/v1beta2"
```

**Comment by viovanov** [comment]
@drnic we're not planning to support 1.16 before 1.0 (December).
Is this a high priority for you?

**Comment by drnic** [comment]
Not a priority. Just a discovery when I was hand crafting a 1.16 cluster.
-- 
Dr Nic Williams
Stark & Wayne LLC
http://starkandwayne.com
+61 437 276 076
twitter @drnic

**Comment by loewenstein** [comment]
@viovanov Did you know [Ephemeral Containers](https://kubernetes.io/docs/concepts/workloads/pods/ephemeral-containers/) are `alpha` in 1.16.
Should we re-prioritize supporting 1.16 for pre-1.0 to get debug containers for problem analysis?

**Comment by manno** [comment]
I believe this was fixed by 57bec03f and 1adb2cd4

**Comment by jkbschmid** [comment]
> I believe this was fixed by #728 and maybe [1adb2cd](https://github.com/cloudfoundry-incubator/cf-operator/commit/1adb2cd419109e0471e2da118c71e725e15f88d7)

Can confirm. We have been using 1.16 for a while now.
